README.md

```diff
@@ -4,3 +4,88 @@
 # AppMap for JetBrains IDEs
 
 Please see [./description.md](./description.md) for a detailed description of the AppMap plugin for JetBrains IDEs.
+
+## Deployment
+
+By default, the plugin ZIP file only bundles the JAR file of the AppMap Java agent.
+
+It's possible to customize settings and/or to bundle AppMap binaries into the plugin ZIP afterward to simplify the
+deployment.
+
+### Customizing Deployment
+
+By modifying the plugin ZIP archive, the deployment can be customized.
+
+The current status of the deployment is shown with action `Tools > AppMap > Plugin Status Report`.
+
+Settings can be customized by adding a `site-config.json` file one of the following locations:
+
+- `./resources/`
+- `./extension/resources/`
+
+The first file found is used.
+
+The file must be a valid JSON file.
+It allows configuring telemetry and auto-update of AppMap binaries.
+By default, no custom telemetry is configured and auto-update is enabled.
+
+Default settings:
+
+```json
+{
+    "appmap.telemetry": null,
+    "appMap.autoUpdateTools": true
```

Comment:
There's inconsistent capitalization in these settings. It should be `appMap` in both.